bokeh/embed.py

```diff
@@ -462,6 +465,20 @@ def _script_for_render_items(docs_json, render_items, websocket_url=None, wrap_s
     else:
         return plot_js
 
+
+def _script_for_notebook_render_items(docs_json, render_items, notebook_comms_target=None,
+                                      wrap_script=True):
+    plot_js = _wrap_in_function(NOTEBOOK_JS.render(
+        docs_json=serialize_json(docs_json),
+        render_items=serialize_json(render_items),
+        comms_target=notebook_comms_target
+    ))
+
+    if wrap_script:
```

Comment:
I don't think you should bother with the wrap_script option. It's very specifically available for `embed.components` which doesn't apply here and as far as I can tell is always going to be `False`.